spec/models/post_spec.rb

```diff
@@ -1,5 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  
+  subject { described_class.new }
+
+  it "is valid with valid attributes" do
+    subject.title = "Anything"
+    subject.body = "Anything body"
+    subject.user = User.create!(email: "anything@io", password: "not_empty", password_confirmation: "not_empty")
+    expect(subject).to be_valid
+  end
+
+  it "is not valid without a title" do
+    post = Post.new(title: nil)
```

Comment:
Here  you would use,

```suggestion
    post = build(:post, title: nil)
```

Reply:
Is it the same or the latter is suggested  more?

Reply:
the green one is the suggested line

Reply:
Oh its like in the controller? Same method?

Reply:
`build` is a method from factorybot, you have to use that

Reply:
Cool.